File: test-suite/src/test/java/io/micronaut/docs/inject/qualifiers/named/V6Engine.java
package io.micronaut.docs.inject.qualifiers.named;

import javax.inject.Singleton;

// tag::class[]
@Singleton
public class V6Engine implements Engine {  // <2>
    public String start() {
        return "Starting V6";
    }

    public int getCylinders() {
        return 6;
    }
}
// end::class[]

File: test-suite-kotlin/src/test/kotlin/io/micronaut/docs/inject/qualifiers/named/V6Engine.kt
package io.micronaut.docs.inject.qualifiers.named

import javax.inject.Singleton

// tag::class[]
@Singleton
class V6Engine : Engine {  // <2>

    override var cylinders: Int = 6

    override fun start(): String {
        return "Starting V6"
    }
}
// end::class[]
